### DISCUSSION TITLE
posts/Blog-Comments/

### DISCUSSION THREAD

**Discussion by giscus** [da-in/da-in.github.io#4] [General]
# posts/Blog-Comments/

댓글 기능을 추가해보자!

https://da-in.github.io/posts/Blog-Comments/

**Comment by da-in** [comment]
테스트 댓글입니다🙂

**Comment by 2choonghyeon** [comment]
안녕하세요. 좋은 글 감사합니다!
repo_id, category_id는 어떻게 얻을 수 있는 건가요?

**Reply by da-in** [reply]
안녕하세요!
repo_id와 category_id는 giscus 생성 화면에서 레포 주소와, 카테고리 옵션을 입력하면,
페이지 하단에 생성된 <script> 태그 속성에서 확인할 수 있습니다!

**Reply by 2choonghyeon** [reply]
감사합니다!! 큰 도움이 되었습니다 :)

**Reply by da-in** [reply]
좋은 개발 되세요🙂

**Comment by pjw74** [comment]
안녕하세요! 혹시 해당 포스트 참고해서 글을 작성해도 괜찮을까요? 
출처 남기겠습니다!

**Reply by da-in** [reply]
네 작성하셔도 됩니다!🙂

**Comment by anveloper** [comment]
구글 검색해서 찾고 있었는데, 체다 글이었네요 ㅋㅋ

**Comment by lamininA1** [comment]
누군가에게 도움이 될까 싶어서 남겨놓습니다. 위 글을 따라해도 comment가 활성화가 안 된다면
`active:` 라고 되어있는 것을 `provider:` 로 변경해 보세요. 저는 잘 해결했습니다.

**Comment by dongseoki** [comment]
저도 이거 읽고 적용해보려고요!, 잘되면 블로그 친구해요~
